### PR TITLE
update css to be more uniform 

### PR DESCRIPTION
**Pull request by Rytiggy**

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -20,51 +22,72 @@ $light-grey-banner: #eeeeee;
...
+  margin: 15px 15px;
+  padding: 20px;
+
+  /* Global */
+
+  h2:hover > .headerlink {
+    opacity: 1;
+  }
+
+  section {
```
Comment (cameronblandford) [inline app/styles/app.scss]: These should be global, not confined to section-detail.

File: app/styles/app.scss [hunk 2]
```diff
@@ -20,51 +22,72 @@ $light-grey-banner: #eeeeee;
...
     padding-top: 30px;
     padding-bottom: 30px;
-}
+  }
 
-section {
+  section {
```
Comment (cameronblandford) [inline app/styles/app.scss]: This should be gotten rid of

File: app/styles/app.scss [hunk 3]
```diff
@@ -20,51 +22,72 @@ $light-grey-banner: #eeeeee;
...
+  .case-number {
+    font-weight: 600;
+    color: #888;
+  }
 
-    h2 {
-        padding: 0;
-        margin-top: 0;
-    }
+  h2 {
```
Comment (cameronblandford) [inline app/styles/app.scss]: This should be global as well

**Comment by cameronblandford** [comment]
All CSS files should have 4 spaces for each soft tab, not 2

File: app/styles/app.scss (hunk 1 above)
Comment (Rytiggy) [reply app/styles/app.scss]: +1

File: app/styles/app.scss (hunk 2 above)
Comment (Rytiggy) [reply app/styles/app.scss]: +1

File: app/styles/app.scss (hunk 3 above)
Comment (Rytiggy) [reply app/styles/app.scss]: +1

File: app/components/meeting-schedule/component.js [hunk 4]
```diff
@@ -84,11 +85,39 @@ export default Ember.Component.extend({
...
+        },
+        applyFilter(){
+            $(".edit-filter-modal").toggleClass("hidden");
+            $(".filter, .filter-remove, .fa-plus, .fa-filter").removeClass("filter-hidden");
+
+        },
+        addFilter(){
+            let filter = this.get('filters');
+            filter.pushObject({id:(filter.length), name: $(`#${event.target.id} :selected`).text() });
+            this.set('filters', filter)
```
Comment (cameronblandford) [inline app/components/meeting-schedule/component.js]: semicolon

File: app/components/meeting-schedule/component.js [hunk 5]
```diff
@@ -84,11 +85,39 @@ export default Ember.Component.extend({
...
+        },
+        removeFilter(){
+            $(event.target).parent().remove()
```
Comment (cameronblandford) [inline app/components/meeting-schedule/component.js]: semicolon

File: app/components/meeting-schedule/component.js [hunk 6]
```diff
@@ -84,11 +85,39 @@ export default Ember.Component.extend({
...
+        },
+        getInput(e){
+            let filter = this.get('filters');
+            if(filter.some(function(o){return o["name"] === e;}) || e.replace(/ /g,'') === ""){
+                return false;
+            }
+            filter.pushObject({id:(filter.length), name: e });
+            this.set('filters', filter)
```
Comment (cameronblandford) [inline app/components/meeting-schedule/component.js]: semicolon

File: app/components/meeting-submit/template.hbs [hunk 7]
```diff
@@ -1,7 +1,5 @@
-<button
-    class='btn btn-primary submit_button'
-    disabled={{if disabled "disabled" null}}
-    {{action "pressButton"}}
->
+<button class='btn btn-primary submit_button' disabled={{if disabled "disabled" null}} {{action "pressButton"}}>
     {{description}}
 </button>
+<br />
```
Comment (cameronblandford) [inline app/components/meeting-submit/template.hbs]: don't include /'s in <br>s

File: app/components/osf-footer/template.hbs [hunk 8]
```diff
@@ -1,16 +1,6 @@
-<style>
```
Comment (cameronblandford) [inline app/components/osf-footer/template.hbs]: :+1:

File: app/controllers/collections/collection/edit.js [hunk 9]
```diff
@@ -17,10 +17,9 @@ export default Ember.Controller.extend({
         }
         return [];
     }),
-    settingsString: Ember.computed('collection.settings', function() {
-        return JSON.stringify(this.get('collection.settings'));
+    settingsString: Ember.computed('model.settings', function() {
```
Comment (cameronblandford) [inline app/controllers/collections/collection/edit.js]: these should be collection right?

File: app/controllers/collections/collection/edit.js [hunk 10]
```diff
@@ -57,14 +56,14 @@ export default Ember.Controller.extend({
     resetModelCache() {
         const collection = this.get('collection');
         return {
-            title: collection.get('title'),
```
Comment (cameronblandford) [inline app/controllers/collections/collection/edit.js]: `collection` is correct here I thought?

File: app/templates/create.hbs [hunk 11]
```diff
@@ -1,62 +1,62 @@
...
-</section>
+    <section class="section">
+        <h4>Add title </h4>
+        {{input class="form-control" value=title placeholder="Enter collection name"}}
+    </section>
 
-<section>
-    <h4>2. Select collection type</h4>
-    {{#each typeList as |type|}}
+    <section class="section">
```
Comment (cameronblandford) [inline app/templates/create.hbs]: why have class="section" here?

File: app/styles/mixins.scss [hunk 12]
```diff
@@ -1,10 +1,10 @@
 @mixin floating {
-  box-shadow: 0 5px 5px -5px rgba(0,0,0,0.3);
+	box-shadow: 0 5px 5px -5px rgba(0,0,0,0.3);
```
Comment (cameronblandford) [inline app/styles/mixins.scss]: spaces not tabs

File: app/templates/create.hbs (hunk 11 above)
Comment (cameronblandford) [reply app/templates/create.hbs]: it would be best to remove or rename this

File: app/templates/collections/collection/add.hbs [hunk 13]
```diff
@@ -1,33 +1,33 @@
 <h1 class="form-banner">{{caxe.workflow.title}}</h1>
 
-<div class="container">
+<div class="form-holder">
```
Comment (cameronblandford) [inline app/templates/collections/collection/add.hbs]: just to make sure - was removing the bootstrapiness from is code intentional?

**Pull request merged**